app/controllers/invitations_controller.rb

```diff
@@ -58,6 +58,12 @@ def after_invite_path_for(resource)
     users_path
   end
 
+  # TODO: remove this method when we're on a version of devise_invitable which
```

Comment:
I see that we can't safely upgrade to it yet, but can we say which version drops the requirement?